ddtrace/profiling/_nogevent.py

```diff
@@ -0,0 +1,65 @@
+# -*- encoding: utf-8 -*-
```

Comment:
lol "_nogevent.py" gotta love that we have to bend over backwards to deal with gevent 🙈 🤦 

I can't really think of a better name... maybe "compat" or "truth" lol.. 🤷

Reply:
`truth` haha.

I had this idea of module for a few weeks, but I was not able to come with a name.

Finally, it hit me. 😆